src/KiwiBoardFirmware_main.cpp

```diff
@@ -247,6 +247,10 @@ void CALLBACK_FUNCTION dry(int id) {
     run(2, &menuDry);
 }
 
+void CALLBACK_FUNCTION test(int id) {

```

Comment:
Lets rename this `motorTest` or similar so that it is less ambiguous

Reply:
Yep - good idea